### PR TITLE
chore(GRO-2166): add min max validation on estimated value of property

### PR DESCRIPTION
**Pull request by WeiLu1**
#### Description
Title says all.

#### Background Context
Part of the bugs found in bug bash for the release of aggregator work.

### CODE REVIEW

**Comment by selina-bot-user** [comment]
@WeiLu1: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr | a9808823a0b5b78f4ec5b9a2b2aff435ee6874a6 | [link](http://dashboard-jx.selinadev.co.uk/namespaces/jx/pipelineruns/finance-ms-lending-api-pr-186-pr-pbxgq) | `/test pr`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **jx-selina-finance-ms-lending-api-pr-186** [here](http://ms-lending-api-pr186.selinadev.co.uk)

File: src/main/java/com/selina/lending/internal/dto/quote/QuickQuotePropertyDetailsDto.java [hunk 1]
```diff
@@ -73,6 +74,7 @@ public class QuickQuotePropertyDetailsDto {
     @Size(min = 2, max = 60)
     private String country;
 
+    @Range(min = 0, max = 99_999_999)
```
Comment (DanielFernandesPTech) [inline src/main/java/com/selina/lending/internal/dto/quote/QuickQuotePropertyDetailsDto.java:77]: The minimum should be 0?

Comment (DanielFernandesPTech) [reply src/main/java/com/selina/lending/internal/dto/quote/QuickQuotePropertyDetailsDto.java:77]: The ticket only have a maximum value. For me make sense a minimum value. I am only concerned if it should be 0 or greater. Can you ask Brendan about it?

Comment (DanielFernandesPTech) [reply src/main/java/com/selina/lending/internal/dto/quote/QuickQuotePropertyDetailsDto.java:77]: However it looks good to me

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **jx-selina-finance-ms-lending-api-pr-186** [here](http://ms-lending-api-pr186.selinadev.co.uk)

**Review by angelo-paiva-ptech — APPROVED** [review]

**Comment by selina-bot-user** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[angelo-paiva-ptech](https://github.com/Selina-Finance/ms-lending-api/pull/186#pullrequestreview-1490981618 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=Selina-Finance%2Fms-lending-api).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/Selina-Finance/ms-lending-api/blob/main/OWNERS)~~ [angelo-paiva-ptech]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by selina-bot-user** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[angelo-paiva-ptech](https://github.com/Selina-Finance/ms-lending-api/pull/186#pullrequestreview-1490981618 "Approved")*, *[DanielFernandesPTech](https://github.com/Selina-Finance/ms-lending-api/pull/186#issuecomment-1600991669 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=Selina-Finance%2Fms-lending-api).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/Selina-Finance/ms-lending-api/blob/main/OWNERS)~~ [DanielFernandesPTech,angelo-paiva-ptech]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **jx-selina-finance-ms-lending-api-pr-186** [here](http://ms-lending-api-pr186.selinadev.co.uk)

**Comment by selina-bot-user** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[angelo-paiva-ptech](https://github.com/Selina-Finance/ms-lending-api/pull/186#pullrequestreview-1490981618 "Approved")*, *[DanielFernandesPTech](https://github.com/Selina-Finance/ms-lending-api/pull/186#issuecomment-1600991669 "Approved")*, *[KastorDriver](https://github.com/Selina-Finance/ms-lending-api/pull/186#issuecomment-1600993484 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=Selina-Finance%2Fms-lending-api).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/Selina-Finance/ms-lending-api/blob/main/OWNERS)~~ [DanielFernandesPTech,KastorDriver,angelo-paiva-ptech]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**